### ISSUE TITLE
auro-popover: add accessibility via keyboard and screen readers

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-popover#10]
# User story

Discovered with work related to https://github.com/AlaskaAirlines/auro-popover/pull/9#pullrequestreview-588107992

## Outline subtasks or tasks

> List specific steps required to deliver this work.

A user that requires the use of the keyboard cannot see the content of the popover. The popover should appear by tabbing to the element alone. No additional keypress needed. 

Once the popover appears, the content of the popover should be announced to the screen reader. This is a follow up to https://github.com/AlaskaAirlines/auro-popover/issues/20

## Definition of “Done” (exit criteria)

> The story is generally “done” when the user can complete the outlined task, but make sure to define what that is.

This story will be considered complete when a popover's content is presented visually and audibly when a user tabs to the element trigger. 

#### Has been tested in all browsers

- [x] Chrome
- [ ] Firefox
- [ ] Edge (79 +)
- [x] Safari
- [ ] Mobile Safari
- [ ] IE11

#### Has been tested for accessibility

- [x] Screenreader
- [ ] Axe
- [x] Keyboard use

#### Performance

- [ ] Chrome Lighthhouse report

#### Auro component features

- [ ] README doc completed with all relative information
- [ ] API documentation created
- [ ] Bundled version tested and documented
- [ ] Examples and use case info created
- [ ] Pages added to the Auro Doc Site

## Audit

https://github.com/AlaskaAirlines/auro/issues/22

**Comment by blackfalcon** [comment]
The resolution for this issue may be in the `inert` feature that @geoffrich recently completed with auro-interruption. 

Please see the last two PRs from @geoff https://github.com/AlaskaAirlines/auro-interruption/pulls?q=is%3Apr+is%3Aclosed

Please consult with @geoffrich for support in adding this functionality to the popover element.
